contract/src/main/kotlin/io/provenance/scope/loan/utility/DataExtensions.kt

```diff
@@ -1,22 +1,32 @@
 package io.provenance.scope.loan.utility
 
-import com.google.protobuf.Message
-import com.google.protobuf.Timestamp
-import io.provenance.scope.util.toInstant
-import tech.figure.util.v1beta1.Checksum
-import tech.figure.util.v1beta1.Date
-import tech.figure.util.v1beta1.Money
-import tech.figure.util.v1beta1.UUID
-import java.time.Instant
+import com.google.protobuf.Message as ProtobufMessage
+import com.google.protobuf.Timestamp as ProtobufTimestamp
+import java.util.UUID as JavaUUID
+import tech.figure.util.v1beta1.Checksum as FigureTechChecksum
+import tech.figure.util.v1beta1.Date as FigureTechDate
+import tech.figure.util.v1beta1.Money as FigureTechMoney
+import tech.figure.util.v1beta1.UUID as FigureTechUUID
```

Comment:
Along the lines of [my previous comment](https://github.com/provenance-io/loan-package-contracts/pull/1/files#r852226738), I wanted to ensure no confusion as to what was being referenced for classes without unique names.